src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,39 @@
+{% set filterableOptions = { projects: [], types: [] } %}
+{% for resource in collections['resources_' + lang] %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("projects", resource.data.projects) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("types", resource.data.type) %}
+{% endfor %}
+<div id="filters" class="filters flow">
+    <h3>{% __ 'filters' %}</h3>
+    <div class="flow">
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'project' %}</legend>
+            {% disclosure 'project' %}
```

Comment:
The `::focus` state of the toggle button is not any different from the default state. Not sure what's happening there.